### ISSUE TITLE
Low specifity on my data

### ISSUE THREAD

**Issue by PelFritz** [ageron/handson-ml2#299]
Hi @ageron ,
i have a question on CNNs. I have a balanced training dataset, but my validation set is way too imbalanced. say out of 600,000 observations, 560,000 comes from class 1 and just 4000 from class 0. Training my network, it has about 78% accuracy on the validation set, a precision of 84% and a specifity of just 15% and its performance on class 0 is poor. My question is does having an imbalance validation set influence optimization of the model parameters? And does such a model make sense eventhough it has a really poor specifity?
